### PR TITLE
Widget for Kyma modules

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: public/extensions/extensions.yaml [hunk 1]
```diff
@@ -1 +1,137 @@
-[]
\ No newline at end of file
+dataSources:
```
Comment (mrCherry97) [inline public/extensions/extensions.yaml:1]: remove this, the rest looks fine

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by mrCherry97 — DISMISSED** [review]

File: docs/extensibility/display-section.md [hunk 2]
```diff
@@ -350,6 +351,32 @@ Labels widgets render all the array or object entries in the `value` or `key-val
...
 <img src="./assets/display-widgets/Labels.png" alt="Example of a Labels widget" width="40%" style="border: 1px solid #D2D5D9">
 
+### ResourceButton
+
+ResourceButton widgets render button that links to Kubernetes resources.
+
+#### Widget-specific parameters
+
+- **icon** - Name of an icon for this button. List of available icons can be found [here](https://sap.github.io/fundamental-react/?path=/docs/component-api-icon--primary). Use string after `--`, e.g. if icon is named `sap-icon--accept` use `accept`.
+- **resource** - To create a hyperlink, Busola needs the name and the kind of the target resource; they must be passed into the **resource** object as property paths in either **\$item** - value extracted using **source**, or **\$root** - the original resource. If the target resource is in a `namespace`, provide **namespace**, **name**, and **kind** properties.
```
Comment (IwonaLanger) [inline docs/extensibility/display-section.md:361]:
```suggestion
- **resource** - To create a hyperlink, Busola needs the name and the kind of the target resource; they must be passed into the **resource** object as property paths in either **\$item** - value extracted using **source**, or **\$root** - the original resource. If the target resource is in a `namespace`, provide **Namespace**, **name**, and **kind** properties.
```

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**